source/_includes/asides/component_navigation.html

```diff
@@ -50,6 +50,13 @@
     <div class='section'>
       Source: <a href='https://github.com/home-assistant/core/tree/dev/homeassistant/components/{{ page.ha_domain }}'>View on GitHub</a>
     </div>
+    {%- if site.data.analytics_data.integrations[page.ha_domain] -%}
+      {% assign percentage = 100 | times: site.data.analytics_data.integrations[page.ha_domain] | divided_by: site.data.analytics_data.reports_integrations | ceil %}
+      <div class='section analytics-data'>
+        <span>Active in {{ percentage }}% Home Assistant installations<i>*</i></span>
+        <i>*Calculated by the <a href="/integrations/analytics">Analytics integration</a></i>
```

Comment:
Maybe we should link this to the big analytics page instead?
```suggestion
        <i>*Calculated by the <a href="/integrations/analytics">Analytics integration</a></i>
```

Reply:
Alternatively, maybe not even mention it?

![image](https://user-images.githubusercontent.com/195327/114269548-18b71680-9a08-11eb-98e6-70a4f53adf49.png)

Just link the statement containing the percentage to the analytics page?

Reply:
Hmmmm that might look ugly too... Not sure myself 🤷